corehq/apps/commtrack/static/commtrack/ko/locations.js

```diff
@@ -6,6 +6,10 @@ $(function() {
 
     model.load(settings);
     ko.applyBindings(model, $('#settings').get(0));
+
+    $("form#settings").on("change input", function() {
```

Comment:
is `"change input"` just some magic jquery thing?  That's pretty handy if so.  I wonder if there's an easy way to just say "disable this button until the form is changed.  I just googled it and found [this](https://stackoverflow.com/questions/13418963/jquery-onchange-function-not-triggering-for-dynamically-created-inputs)  are you sure it doesn't need to be:
```javascript
$("form#settings").on("change", "input", fn);
```